### ISSUE TITLE
Disable Nextjs i18n by default.

### ISSUE THREAD

**Issue by promer94** [shuding/nextra#626]
Maybe we should disbale Nextjs i18n because it does not work with next export

**Comment by shuding** [comment]
That makes sense, I never tried Nextra with `next export` 🥲

**Comment by samuelgoff** [comment]
hey @shuding, same boat here. It keeps failing on export (`Error: i18n support is not compatible with next export`), even when I [disable automatic locale detection](https://nextjs.org/docs/advanced-features/i18n-routing#disabling-automatic-locale-detection).

Is there something else I need to be doing in order to make this work with `next export`? If not, then simply disabling nextjs i18 by default alone won't fix this issue.

**Comment by shuding** [comment]
Yep, something needs to be changed in Nextra's code to make it work @samuelgoff

**Comment by B2o5T** [comment]
Can fix it but only the next week

**Comment by samuelgoff** [comment]
@shuding by the way, I filed an [issue](https://github.com/vercel/next.js/issues/39269) against nextjs because following the docs to disable i18n alone produces errors. That said, while creating the [minimal reproduction](https://stackblitz.com/edit/vercel-next-js-hjzwyk?file=next.config.js) on stackblitz, I was able to run `next export` successfully -- as long as I removed i18n config from the `next.config.js`. My guess is the nextra config populates this by default, so nextjs always thinks it's enabled.
